src/transaction/utils.ts

```diff
@@ -2,6 +2,7 @@ import BigNumber from 'bignumber.js'
 import * as common from '../common'
 import {Memo} from '../common/types/objects'
 import {Instructions, Prepare, TransactionJSON} from './types'
+import { toRippledAmount } from '../common'
```

Comment:
Updating the linter (if needed) can be done separately, but for now we could just remove the spaces between { and } so that it matches the existing code:
```suggestion
import {toRippledAmount} from '../common'
```